src/sema/types.rs

```diff
@@ -1056,6 +1056,65 @@ impl Type {
         }
     }
 
+    /// Retrieve the alignment for each type, if it is a struct member.
+    /// The representation outside a struct varies a little, for arrays are always reference types
+    /// outside structs.
```

Comment:
I don't understand what this comment means. Are you talking about dynamic arrays?